test/hga/HGA_Regression.ipynb

```diff
@@ -49,15 +63,20 @@
     "from harmony import BBox, Client, Collection, Environment, Request\n",
     "\n",
     "from test_utilities import (check_request_output, clean_test_artefacts,\n",
-    "                            make_request_and_download_result, print_success)"
+    "                            get_test_granule_id, make_request_and_download_result,\n",
+    "                            print_success)"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "1fbb7835",
    "metadata": {},
    "source": [
-    "### Set up environment dependent variables:"
+    "### Set up environment dependent variables:\n",
+    "\n",
+    "This includes the Harmony `Client` object and `Collection` objects for each of the collections for which there are regression tests. The local, SIT and UAT Harmony instances all utilise resources from CMR UAT, meaning any non-production environment will use the same resources.\n",
+    "\n",
+    "When adding a production entry to the dictionary below, the collection instances can be included directly in the dictionary, as they do not need to be shared."
```

Comment:
"when adding a production entry" huh?  I'm not sure exactly what you're getting at. Is this a note for when someone adds a production regression test, then they can just add that whole production dictionary?  I think that should be clearer if so, if not, I don't  know what this means.

Reply:
Could you add a  TBD  for production in the collection_data dictionary and just raise an error or something?

Reply:
Or even just remove that last comment in there.  Whoever is doing the production work would probably figure it out.  or

```
When adding a production entry to the dictionary below, the collection instances can be included directly in the production dictionary entry, as they do not need to be shared.
```